src/app/campaign-details/campaign-details.component.html

```diff
@@ -5,23 +5,28 @@ <h2>Banner Image here!</h2>
 <div class="details" *ngIf="campaign">
 	<div>
 		<h2>{{campaign.title}}</h2>
-    <h3 *ngIf="campaign.championName">Championed by {{campaign.championName}}</h3>
+    	<h3 *ngIf="campaign.championName">Championed by {{campaign.championName}}</h3>
 		<h4 *ngIf="campaign.isMatched">Matched Campaign!</h4>
 		<h4 *ngIf="!campaign.isMatched">Regular campaign</h4>
 	</div>
 
 	<div>
 		<h3>Campaign Summary</h3>
-    <p>{{campaign.summary}}</p>
-    <p>Status: <strong>{{ campaign.status }}</strong></p>
-    <p *ngIf="campaign.status == 'Active'">{{ campaign.endDate | timeLeft }} left</p>
+		<p>{{campaign.summary}}</p>
+		<p>Status: <strong>{{ campaign.status }}</strong></p>
+		<p *ngIf="campaign.status == 'Active'">{{ campaign.endDate | timeLeft }} left</p>
 	</div>
 
 	<div>
 		<h3>Campaign Targets</h3>
 		<p *ngIf="campaign.target > 0">Campaign Target: {{ campaign.target | currency : '£' : 'symbol' : '1.0-0' }}</p>
-    <p *ngIf="campaign.amountRaised > 0">Amount Raised: {{ campaign.amountRaised | currency : '£' : 'symbol' : '1.0-0' }}</p>
-    <p>Donations: {{ campaign.donationCount }}</p>
+		<p *ngIf="campaign.amountRaised > 0">Amount Raised: {{ campaign.amountRaised | currency : '£' : 'symbol' : '1.0-0' }}</p>
+		<p>Donations: {{ campaign.donationCount }}</p>
+	</div>
+
+	<div *ngIf="campaign.alternativeFundUse != null">
```

Comment:
@tbg-jasonfung with the current proposed model change this is never actually `null` (the type is not nullable), so would probably make more sense to test for either `campaign.alternativeFundUse` (is it 'truth-y' in general?) or `campaign.alternativeFundUse.length > 0`

Reply:
@NoelLH Interestingly, I swapped to use `.length` check but its throwing this error:

![image](https://user-images.githubusercontent.com/47176130/64526265-be6d4580-d2fa-11e9-9ef1-d120d28c245a.png)

Reply:
@tbg-jasonfung in that case the API and the client model are not consistent. We should probably make the property nullable in the model (similar to optional fields on `Donation`) if the API is returning a null value or omitting the property.

The truth-i-ness check option will work either way I think, but we should make our types accurate in order to benefit from the checks built into TypeScript.